src/getExtendedPublicKey.c

```diff
@@ -100,19 +70,7 @@ void io_exchange_address()
 
 	G_io_apdu_buffer[tx++] = 32;
```

Comment:
Please, alias this to a buffer, later we might want to refactor the rest of the code to use
io_send_buf helper function from io.h

Reply:
Hmm, why was this marked as resolved?